scripts/dts/extract/globals.py

```diff
@@ -75,6 +75,21 @@ def get_aliases(root):
         if reduced[k].get('alt_name', None) is not None:
             aliases[k].append(reduced[k]['alt_name'])
 
+def get_compats(node_address):
```

Comment:
Another difference with get_compat is that this one won't get parent compat if node has no compat.
Can it be renamed to get_node_compats to make it  a bit clearer?

Reply:
Changed to get_node_compats